package.json

```diff
@@ -63,7 +63,7 @@
     "@nteract/markdown": "^4.0.0",
     "@nteract/mathjax": "^4.0.0",
     "@nteract/outputs": "^2.1.5",
-    "@nteract/transform-plotly": "^5.0.0",
+    "@nteract/plotly": "^1.48.3",
```

Comment:
So mainly this dependency is needed only for specs ?
If so, let's move this into `devDependency`

Reply:
(For better loading time)

Reply:
@aviatesk no it's needed for both

Reply:
You mean it's implicitly imported somewhere ?

Reply:
Inside result-view/plotly.js

Reply:
Ah I missed it. I see.